x/concentrated-liquidity/spread_rewards.go

```diff
@@ -317,3 +347,62 @@ func updatePositionToInitValuePlusGrowthOutside(accumulator *accum.AccumulatorOb
 	}
 	return nil
 }
+
+// scaleDownSpreadRewardAmount scales down the spread reward amount by the scaling factor.
+// This method differs from scaleDownIncentiveAmount in that it not only returns the scaled amount, but also the truncated decimal, which is dust
+// that is added back to the global accumulator.
+func scaleDownSpreadRewardAmount(incentiveAmount osmomath.Int, scalingFactor osmomath.Dec) (scaledAmount osmomath.Int, truncatedDec osmomath.Dec) {
+	scaledDec := incentiveAmount.ToLegacyDec().QuoTruncate(scalingFactor)
+
+	// Scaled down amount, which is used to distribute to user
+	scaledAmount = scaledDec.TruncateInt()
+
+	// Truncated decimal, which is dust that is added back to the global accumulator
+	truncatedDec = scaledDec.Sub(scaledAmount.ToLegacyDec())
```

Comment:
Does it matter that much?

IMO the added complexity and performance hit from math operations might not be worth it

Reply:
https://github.com/osmosis-labs/osmosis/blob/0b77070a7ecafe73f9b0183b8730891395d57c22/x/concentrated-liquidity/spread_rewards.go#L257-L290

Is your recommendation then, use forfeitedDustScaled when the scaling factor is 1, otherwise throw away dust?

Reply:
Done here https://github.com/osmosis-labs/osmosis/pull/7876/commits/c1cdbbfcc7e2f93f86168777ad7ba8d0db16e02f

Would like an ack prior to merging though so I don't just slip this in

Reply:
CC @PaddyMc or @AlpinYukseloglu for an ack prior to me merging